### DISCUSSION TITLE
Stockfish for Windows XP 32 bit

### DISCUSSION THREAD

**Discussion by Batu-64** [official-stockfish/Stockfish#5638] [General]
Why are you depriving Windows XP users of this engine? It shouldn't be too difficult for you to release this!

**Comment by Disservin** [comment]
32 bit builds were last distributed with Stockfish 14.1 . Since then we have moved to compiling the binaries in our CI which doesn't have 32 bit support. It is also not really something users are realistically interested other than for historic purposes, in which case just compile this yourself.

**Comment by Sopel97** [comment]
Windows XP does not have the necessary WinAPI function to facilitate Stockfish. Sorry, it won't happen.

**Reply by Batu-64** [reply]
So, what WinAPIs does Windows XP not have? How is Stockfish 14.1 able to run on Windows XP without using these WinAPIs?

**Reply by Sopel97** [reply]
the bulk of numa.h, and it would be a mess to special case older windows and would result in different behaviour

**Reply by vdbergh** [reply]
Why not disable NUMA on older systems?

**Reply by Sopel97** [reply]
patches welcome

**Reply by vondele** [reply]
I wouldn't even say that. Adding code to deal with a very old OS is a maintenance burden, and adds more code that can't be tested in CI. Only when there is a clear benefit to the project should we do such things.

XP went out of support in 2009 and out of extended support in 2014. A user that is stuck to such an old OS should accept that the latest software does no longer work.
